PyStemmusScope/forcing_io.py

```diff
@@ -0,0 +1,138 @@
+import os
+import numpy as np
+import xarray as xr
+
+
+def _calculate_ea(t_air_celcius, rh):
+    """Internal function that calculates the actual vapour pressure (kPa) from the
+    air temperature (degree Celcius) and relative humidity (%)
```

Comment:
Can we add a link as a reference for this equation? It is a version of [Tetens_equation ](https://en.wikipedia.org/wiki/Tetens_equation), but I couldn't find a reference for the equation used here.

Reply:
I am also puzzled why they used this equation. I can not find an equation in this form, but is a 1:1 implementation from the Matlab code they use.

While they call it "ea", the values are indeed at least a factor 10 off compared to using a correct equation for the saturation vapor pressure.

Reply:
I made issue https://github.com/EcoExtreML/STEMMUS_SCOPE/issues/62